### PR TITLE
community: update meilisearch vectorstore for compatibility with Meilisearch v1.11

### PR DESCRIPTION
**Pull request by CaroFG**
Thank you for contributing to LangChain!

- [x] **PR title**: "package: description"

  - Where "package" is whichever of langchain, community, core, etc. is being modified. Use "docs: ..." for purely docs changes, "infra: ..." for CI changes.
  - Example: "community: add foobar LLM"

- [x] **PR message**: 
    
    - Renames `embedder_name` into `embedder`for consistency with Meilisearch naming convention. As of Meilisearch v1.11 `embedder` parameter (referred to as as `embedder_name` in this package) is mandatory 
    - Makes the above mentioned `embedder` parameter mandatory to ensure compatibility with [Meilisearch v1.11](https://github.com/meilisearch/meilisearch/releases/tag/v1.11.0) 
   - Adds the `embedder` parameter to the Meilisearch vectorstore initialization to avoid potential signature incompatibility error with with supertype "VectorStore"
   - Removes the embedder settings update: this update modifies the handling of the embedders parameter to prevent unintended clearing of embedder settings. 
   - Updates tests
   - Updates documentation

    

- [ ] **Add tests and docs**: If you're adding a new integration, please include
  1. a test for the integration, preferably unit tests that do not rely on network access,
  2. an example notebook showing its use. It lives in `docs/docs/integrations` directory.

- [x] **Lint and test**: Run `make format`, `make lint` and `make test` from the root of the package(s) you've modified. See contribution guidelines for more: https://python.langchain.com/docs/contributing/

Additional guidelines:
- Make sure optional dependencies are imported within a function.
- Please do not add dependencies to pyproject.toml files (even optional ones) unless they are required for unit tests.
- Most PRs should not touch more than one package.
- Changes should be backwards compatible.
- If you are adding something to community, do not re-import it in langchain.

If no one reviews your PR within a few days, please @-mention one of baskaryan, efriis, eyurtsev, ccurme, vbarda, hwchase17.

### CODE REVIEW

**Comment by CaroFG** [comment]
Hi! It's already been 2 weeks since the PR was created, I am tagging @baskaryan to make sure you don't forget about this 🙏

File: libs/community/langchain_community/vectorstores/meilisearch.py [hunk 1]
```diff
@@ -65,31 +65,25 @@ class Meilisearch(VectorStore):
...
-                embedders=embedders,
+                embedder='custom', # Name to identify the embedder object in Meilisearch
                 client=client,
                 index_name='langchain_demo',
                 text_key='text')
     """
 
     def __init__(
         self,
+        embedder: str,
```
Comment (efriis) [inline libs/community/langchain_community/vectorstores/meilisearch.py:79]: breaking change

File: libs/community/langchain_community/vectorstores/meilisearch.py [hunk 2]
```diff
@@ -65,31 +65,25 @@ class Meilisearch(VectorStore):
...
         embedding: Embeddings,
         client: Optional[Client] = None,
         url: Optional[str] = None,
         api_key: Optional[str] = None,
         index_name: str = "langchain-demo",
         text_key: str = "text",
         metadata_key: str = "metadata",
-        *,
-        embedders: Optional[Dict[str, Any]] = None,
```
Comment (efriis) [inline libs/community/langchain_community/vectorstores/meilisearch.py:92]: leave this

**Review by efriis — COMMENTED** [review]
hey! this is a breaking change through and through so can't really merge as-is. would recommend publishing a separate langchain-melisearch package that you can version control (including breaking version bumps) separately

here's the guide: https://python.langchain.com/docs/contributing/how_to/integrations/

**Pull request closed without merging**